src/App.js

```diff
@@ -51,6 +57,8 @@ class App extends Component {
                key={item.id}            // Key - to help React with performance
 
                // Additional props (event callbacks, etc.) can be passed here
+               type={item.type}
+               onItemClickedCallback={this.onItemClickedCallback}
```

Comment:
It's best practice to name the prop with `Callback` and the function with a meaningful name to what is does.